release-notes/1.1/1.1.0-preview2.md

```diff
@@ -0,0 +1,79 @@
+- Misleading ADP_OpenReaderExists exception message on MARS-disabled Sql Connection when incorrectly doing parallel requests [#82](https://github.com/dotnet/SqlClient/issues/82)
+- SqlClient ManualTest MARSSyncTimeoutTest fails in managed mode [#108](https://github.com/dotnet/SqlClient/issues/108)
+- System.Data.SqlClient.SqlInternalConnectionTds constructor purges original call stack when re-throwing an exception [#100](https://github.com/dotnet/SqlClient/issues/100)
+- InvalidOperationException(SqlException) on SqlBulkCopy [#221](https://github.com/dotnet/SqlClient/issues/221)
+- Exception message grammar: "An SqlParameter [...] is not contained by this SqlParameterCollection" [#159](https://github.com/dotnet/SqlClient/issues/159)
+- Fixing incorrect event id and opcode for the SqlEventSource [#241](https://github.com/dotnet/SqlClient/pull/241)
+
+### Changes
+
+- Update dependency to Microsoft.Data.SqlClient.SNI v1.1.0 [#276](https://github.com/dotnet/SqlClient/pull/276)
+- Update minimum NuGet client version to 5.0 to support buildTransitive [#274](https://github.com/dotnet/SqlClient/pull/274)
+- Correct timeout remarks for async command methods [#264](https://github.com/dotnet/SqlClient/pull/264)
+- Improve SqlBulkCopy truncation error message [#256](https://github.com/dotnet/SqlClient/issues/256)
+- Intellisense tooltip for SqlCommand's CommandTimeout doesn't describe units [#33](https://github.com/dotnet/SqlClient/issues/33)
+- Enable SQL Command text for non-stored procs in EventSource events for .NET Framework [242](https://github.com/dotnet/SqlClient/pull/242)
+- Many test changes to support a public CI
+
+### New Features
+
+New features over the 1.0 release of Microsoft.Data.SqlClient.
+
+- [Always Encrypted with secure enclaves](#always-encrypted-with-secure-enclaves) - Always Encrypted is available starting in Microsoft SQL Server 2016. Secure enclaves are available starting in Microsoft SQL Server 2019.
+
+## Target Platform Support
+
+- .NET Framework 4.6+
+- .NET Core 2.1+ (Windows x86, Windows x64, Linux, macOS)
+- .NET Standard 2.0+ (Windows x86, Windows x64, Linux, macOS)
+
+### Dependencies
+
+#### .NET Framework
+
+- System.Data.Common 4.3.0
+- Microsoft.Data.SqlClient.SNI 1.1.0
+- Microsoft.Identity.Client 3.0.8
+
+#### .NET Core
+
+- Microsoft.Win32.Registry 4.5.0
+- runtime.native.System.Data.SqlClient.sni 4.4.0
+- System.Security.Principal.Windows 4.5.0
+- System.Text.Encoding.CodePages 4.5.0
+- System.Configuration.ConfigurationManager 4.5.0
+- Microsoft.Identity.Client 3.0.8
+
+#### .NET Standard
+
+- Microsoft.Win32.Registry 4.5.0
+- runtime.native.System.Data.SqlClient.sni 4.4.0
+- System.Buffers 4.4.0
+- System.Diagnostics.DiagnosticSource 4.5.0
+- System.Memory 4.5.1
+- System.Security.Principal.Windows 4.5.0
+- System.Text.Encoding.CodePages 4.5.0
+- System.Configuration.ConfigurationManager 4.5.0
+- Microsoft.Identity.Client 3.0.8
+
+### Always Encrypted with secure enclaves
+
```

Comment:
Probably we need to add how to use the enclave feature by specifying `Attestation Protocol` and `Enclave Attestation Url` in the connection string.
e.g. 
`Attestation Protocol = HGS; Enclave Attestation Url = <attestation_url_for_HGS>`
`Attestation Protocol = AAS; Enclave Attestation Url = <attestation_url_for_AAS>`

Reply:
Done